frontend/stories/components/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from "@storybook/react";
+import { PROCESS_CRUMBS, RESEARCH_CRUMBS } from "src/constants/breadcrumbs";
+
+import Breadcrumbs from "src/components/Breadcrumbs";
+
+const meta: Meta<typeof Breadcrumbs> = {
+  title: "Components/Breadcrumbs",
+  component: Breadcrumbs,
+};
+export default meta;
+
+export const Home = {
```

Comment:
I don't think we'd ever show breadcrumbs on root. Okay to show how it'd work if we did tho.

Reply:
Yeah, I wasn't sure what to use for the default story. I could change it to be either Research or Process 🤷, let me know if you have a preference

Reply:
Probably fine as-is. We can revisit in the future as our route/page requirements become more complex.